cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableDefinition.java

```diff
@@ -88,7 +92,10 @@ private HTableDescriptor getHTableDescriptor(DatasetSpecification spec) throws I
 
     final HTableDescriptor tableDescriptor = new HTableDescriptor(tableName);
     tableDescriptor.addFamily(columnDescriptor);
-    tableDescriptor.addCoprocessor(hBaseTableUtil.getIncrementHandlerClassForVersion().getName());
+    AbstractHBaseDataSetAdmin.CoprocessorJar cpJar =
+      HBaseOrderedTableAdmin.createCoprocessorJarInternal(conf, locationFactory, hBaseTableUtil, true);
+    tableDescriptor.addCoprocessor(hBaseTableUtil.getIncrementHandlerClassForVersion().getName(),
```

Comment:
This is a bit hacky, as it's always including the DefaultTransactionProcessor coprocessor in the jar as well, and just not using it in the table configuration.  I suppose this is okay as a short term fix, but do we have a JIRA to unify HTableDatasetAdmin and HBaseOrderedTableAdmin?  That seems like the right way to handle this, instead of leaking this into HBaseMetricsTableDefinition.